nengo_loihi/hardware/api.py

```diff
@@ -0,0 +1,370 @@
+from __future__ import division
```

Comment:
Yeah I'm not sure what to do about the division of things in the `hardware` folder... right now it seems like `builder.py` has build things, and `api.py` has things that the builder needs, so they should kind of be in one file, but that file would be too big so it should be split up. I'm not super sure about the name `api.py` though, who would be using this API? Maybe we could break it up into a file with `Board`, `Chip`, and `Core`, maybe named `physical.py` or `core.py` or something like that, then `profiles.py` for the profiles. Then I'm not sure about `LoihiSpikeInput`, maybe that lives in `builder.py` ?

Reply:
`nxsdk_objects.py`